From 462bceaa166eec0518567c167e69d87d7fc343d1 Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Tue, 7 Nov 2017 19:33:55 -0500
Subject: [PATCH 4243/5855] drm/amdgpu: Remove unused dc_stream from
 amdgpu_crtc

It's no longer used. In fact, there is no more dc_stream object.

Change-Id: If9726d566544d306c8d5208a2820a30d566d6cc1
Signed-off-by: Harry Wentland <harry.wentland@amd.com>
Reviewed-by: Andrey Grodzovsky <andrey.grodzovsky@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_mode.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_mode.h b/drivers/gpu/drm/amd/amdgpu/amdgpu_mode.h
index 3ad868d..3c8739b 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_mode.h
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_mode.h
@@ -441,7 +441,6 @@ struct amdgpu_crtc {
 	uint32_t flip_flags;
 
 	/* After Set Mode stream will be non-NULL */
-	const struct dc_stream_state *stream;
 	struct drm_pending_vblank_event *event;
 };
 
-- 
2.7.4

